src/main/java/com/mixpanel/android/viewcrawler/ViewCrawler.java

```diff
@@ -725,24 +720,24 @@ private void handleEditorBindingsCleared(JSONObject clearMessage) {
                 Log.e(LOGTAG, "Bad clear request received", e);
             }
 
-            updateEditState();
+            applyVariantsAndEventBindings();
         }
 
         private void handleEditorTweaksReceived(JSONObject tweaksMessage) {
             try {
+                mEditorTweaks.clear();
```

Comment:
I wanted to stay as close as I could to the old, known good editor behavior, which was to apply only the new inbound tweaks, and rely on the persistence of the mTweaks that are already applied.